### PR TITLE
Add option to delete images

### PR DESCRIPTION
**Pull request by Hiw**

### CODE REVIEW

**Comment by Hiw** [comment]
[Issue 7: Delete Image](https://trello.com/c/1Dk7qorG/368-issue-7-delete-image)

File: test/controllers/images_controller_test.rb [hunk 1]
```diff
@@ -24,6 +24,22 @@ class ImagesControllerTest < ActionDispatch::IntegrationTest
...
+  test 'should destroy image' do
+    @image = Image.create!(url: @image_url)
+
+    assert_difference 'Image.count', -1 do
+      delete image_url @image
+    end
+
+    assert_redirected_to images_path
+
+    assert_no_difference 'Image.count' do
```
Comment (eric199wen) [inline test/controllers/images_controller_test.rb:36]: It would be better if we can put it in another test.

**Review by eric199wen — APPROVED** [review]
I left one comment about improving test, however i think we can still merge it.

File: test/controllers/images_controller_test.rb [hunk 2]
```diff
@@ -24,6 +24,27 @@ class ImagesControllerTest < ActionDispatch::IntegrationTest
...
+    @image = Image.create!(url: @image_url)
+
+    assert_difference 'Image.count', -1 do
+      delete image_url @image
+    end
+
+    assert_redirected_to images_path
+  end
+
+  test 'should destroy iage that doesn\'t exist' do
```
Comment (eric199wen) [inline test/controllers/images_controller_test.rb:37]: There is a typo here.

File: test/controllers/images_controller_test.rb [hunk 3]
```diff
@@ -24,6 +24,27 @@ class ImagesControllerTest < ActionDispatch::IntegrationTest
...
+    @image = Image.new(url: @image_url)
+    @image.id = 42
```
Comment (eric199wen) [inline test/controllers/images_controller_test.rb:39]: Why do we need to set `id`?

File: test/controllers/images_controller_test.rb [hunk 4]
```diff
@@ -24,6 +24,27 @@ class ImagesControllerTest < ActionDispatch::IntegrationTest
...
+
+    assert_no_difference 'Image.count' do
+      delete image_url @image
```
Comment (eric199wen) [inline test/controllers/images_controller_test.rb:42]: Alternatively, we can just provide a non-existing id as param here.

**Review by eric199wen — APPROVED** [review]

**Pull request merged**